Add OWNERS file for dashboard

wbuchwalter opened the pull request:
fix #443.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/kubeflow/tf-operator/446)
<!-- Reviewable:end -->

jlewi commented on dashboard/OWNERS:
```diff
@@ -0,0 +1,14 @@
+approvers:
+  - jlewi
```
Please remove me. I'm already a root approver and I don't have special knowledge of the Dashboard.

Also do you want to be listed as a root approver
https://github.com/kubeflow/tf-operator/blob/master/OWNERS#L13

I know you have a different role but it might make more sense to have you focus reviewing the UI
and then also kubeflow/examples

jlewi commented on dashboard/OWNERS:
```diff
@@ -0,0 +1,14 @@
+  - jlewi
+  - wbuchwalter
+  - Jimexist
+reviewers:
```
Remove all of these reviewers.
Approvers shouldn't also be listed as reviewers.

None of the non-approvers listed here have special familiarity with react/javascript. 
I sent a request yesterday for more people with javascript/react experience to volunteer but we'll see.

You could add @swiftdiaries as a reviewer. He's working on the central UI.

wbuchwalter commented:
@jlewi per your comment:
* Added myself as a root approver.
* Added only @Jimexist as an approver in the dashboard dir
* Added @swiftdiaries as a reviewer in the dashboard dir
* Cleaned the root OWNERS to remove the reviewers that were already approvers

jlewi commented:
/lgtm
/approve
